cmd/gitannex/run-git-annex-testremote.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+#
+# End-to-end tests for garrgoyle. This script runs the `git-annex testremote`
+# suite against garrgoyle in an ephemeral git-annex repo.
+#
+# Assumptions:
+#
+#   * This system has an rclone remote configured named "garrgoyle-test-remote".
+#
+#   * If it uses rclone's "local" backend, /tmp/garrgoyle-test-remote exists.
```

Comment:
best to avoid any assumptions or otherwise would require per location where you would intend to run this test to set it up somehow etc. What about the approach we have in git-annex-remote-rclone to use temporary HOME with all needed setup? checkout 

https://github.com/git-annex-remote-rclone/git-annex-remote-rclone/blob/master/tests/all-in-one.sh#L22

which might also give you ideas on additional (it is never enough!!) testing etc.

Reply:
@yarikoptic Very clever! In the interest of unblocking the PR, I'll punt on improving the test script for now, but this is the very next thing on my TODO list for the project. I really appreciate you taking a look at the PR.

Reply:
Traveling ATM, can't promise speedy review over weekend, but would be happy to help in general.